tests/sweep_framework/sweeps/ccl/line_all_gather.py

```diff
@@ -65,7 +65,9 @@ def mesh_device_fixture():
     assert ttnn.get_num_devices() >= 8, "Not T3000!"
     device_ids = ttnn.get_t3k_physical_device_ids_ring()
     num_devices_requested = len(device_ids)
-    mesh_device = ttnn.open_mesh_device(ttnn.MeshShape(1, num_devices_requested), device_ids[:num_devices_requested])
+    mesh_device = ttnn.open_mesh_device(
+        ttnn.MeshShape(1, num_devices_requested), physical_device_ids=device_ids[:num_devices_requested]
```

Comment:
This does not guarantee that the devices are connected in a line (if that's what you're testing). Please do this instead:

`mesh_device = ttnn.open_mesh_device(ttnn.MeshShape(1, num_devices_requested), mesh_type=ttnn.MeshType.Line)`

Reply:
Updated

Reply:
excellent, thanks!